app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -678,6 +678,15 @@ const ContentForm: React.FC<IContentFormProps> = ({
                                       }
                                     />
                                   </div>
+                                  <Checkbox
```

Comment:
Why not just use the `<FormikCheckbox/>`?

Reply:
this has been updated. thanks!